Competition2018/src/main/java/competition/subsystems/elevator/commands/MoveElevatorToHeightAndHoldCommand.java

```diff
@@ -0,0 +1,59 @@
+package competition.subsystems.elevator.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.elevator.ElevatorSubsystem;
+import edu.wpi.first.wpilibj.Timer;
+import xbot.common.command.BaseCommand;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class MoveElevatorToHeightAndHoldCommand extends BaseCommand {
+    private ElevatorSubsystem elevator;
+
+    public DoubleProperty errorThreshold;
+    public DoubleProperty stableTimeRequired;
+
+    double initialTime;
+    public int target;
+
+    boolean previouslyAtTarget;
+
+    @Inject
+    public MoveElevatorToHeightAndHoldCommand(ElevatorSubsystem subsystem, XPropertyManager propMan) {
```

Comment:
One more rename (Kaelin's suggestion was really good) to Stabilize, since that is a great descriptor of when it is done.

`MoveElevatorToHeightAndStabilizeCommand`